### ISSUE TITLE
All animated stickers replaces with only accidental one

### ISSUE THREAD

**Issue by adrianopol** [telegramdesktop/tdesktop#6685]
Subj. Probably, it is the first cached sticker.

If I am not mistaken, it was OK in 1.8.8, but bugged in 1.8.9. Now I am on 1.8.15, the bug is still there.

### Configuration
OS: Gentoo Linux
Compiler: sys-devel/gcc-8.3.0
Ebuild: https://gpo.zugaina.org/AJAX/Ebuild/43959296/View (PV=1.8.15) ; all files: https://data.gpo.zugaina.org/4nykey/net-im/telegram-desktop/
tdesktop version: 1.8.15

![Screenshot_2019-10-16_14-04-46](https://user-images.githubusercontent.com/4029800/66914581-c8801380-f01f-11e9-9ff6-e63deae2d6b9.png)
![Screenshot_2019-10-16_14-05-06](https://user-images.githubusercontent.com/4029800/66914584-c9b14080-f01f-11e9-88a5-592efcaccd8b.png)
![Screenshot_2019-10-16_14-05-22](https://user-images.githubusercontent.com/4029800/66914585-c9b14080-f01f-11e9-84eb-6fc3bca52242.png)

**Comment by Johnnynator** [comment]
Recompile rlottie without caching support should fix this and clearing the telegram sticker cache.

**Comment by adrianopol** [comment]
No, just cache cleaning it not enough. Moreover, I have permanently deleted all client-related files, it didn't help.
Will try to rebuild rlottie without cache later, and report the results, thanks.

**Comment by Johnnynator** [comment]
> Will try to rebuild rlottie without cache later, and report the results, thanks.

This was only needed for older versions of Telegram, your issue was that you did run a version once that didn't have the current behavior, so you had a "corrupted" telegram sticker cache.

**Comment by adrianopol** [comment]
```
find ~ -iname '*telegram*' -o -iname '*tdesktop*'
.../.local/share/TelegramDesktop
```

@Johnnynator, I have killed the app, removed `~/.local/share/TelegramDesktop`, and started the app again. I am in doubt if there is any other cache files.

**Comment by adrianopol** [comment]
Fixed in 1.9.1 (uses newer rlottie version): https://gpo.zugaina.org/net-im/telegram-desktop, gentoo-zh overlay.
